app/server/views/extend-deadline/confirmation-form.html

```diff
@@ -0,0 +1,38 @@
+<form action="/extend-deadline" method="post">
+  <div class="govuk-form-group {{ "govuk-form-group--error" if error }}">
+    <fieldset class="govuk-fieldset" aria-describedby="extend-deadline-hint">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+        <h1 class="govuk-fieldset__heading">
+          {{ i18n.extendDeadline.header }}
+        </h1>
+      </legend>
+      <p id="extend-deadline-hint">
+        {{ i18n.extendDeadline.hint }}
+      </p>
+      <span class="govuk-fieldset__legend govuk-fieldset__legend--m">
+        {{ i18n.extendDeadline.subheader }}
+      </span>
+      <div class="govuk-radios govuk-radios--inline">
```

Comment:
Is there a govuk-frontend nunjucks macro for this?

Reply:
Unfortunately our design doesn't quite fit the nunjucks macro

Reply:
Ah right, that was the case for the summary/details elements too, pretty annoying.